jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/BuildTarTask.java

```diff
@@ -80,11 +80,7 @@ public void setTargetImage(String targetImage) {
   public FileCollection getInputFiles() {
```

Comment:
The javadoc makes it sound like only the classes dependencies matter, and there's no mention of the extras.

Reply:
fixed.